irida_staramr_results/validate.py

```diff
@@ -0,0 +1,57 @@
+import sys
+import logging
+import getpass
+import time
+
+from irida_staramr_results import util
+
+
+def user_credentials(username, password):
+    if username is None:
+        username = input("Enter your IRIDA username: \n")
+    if password is None:
+        print("Enter your IRIDA password: ")
+        password = getpass.getpass()
+
+    return {"username": username,
+            "password": password}
+
+
+def output_file_name(file_name):
+    if file_name.endswith(".xlsx"):
+        file_name = file_name[:-len(".xlsx")]
+
+    return file_name
+
+
+def date_range(from_date, to_date):
+    """
+    Sets up FROM and TO date values in unix timestamp.
+    :param from_date:
+    :param to_date:
+    :return:
+    """
+
+    if from_date is None:
+        from_date = 0
+    else:
+        from_date = util.local_to_timestamp(from_date)
+
+    if to_date is None:
```

Comment:
Can you please add some line by line comments for what is happening in this function. You and I both know how the `time` module works and the conversions that are happening here, but the next developer who could work on it might not.

Reply:
Comments added.